### DISCUSSION TITLE
Integrate the experiences of this project into weltenwort/home-assistant-rct-power-integration

### DISCUSSION THREAD

**Discussion by do-gooder** [do-gooder/rctpower_writesupport#13] [Ideas]
I am not the right person to integrate the experiences of this project into [weltenwort/home-assistant-rct-power-integration](https://github.com/weltenwort/home-assistant-rct-power-integration).
I'm sure there are people here who find this easier than I do. My programming skills are not good enough for this.

Would this make sense?
Who could implement it?

**Comment by sektorenkoppler** [comment]
That would be really nice.
